src/app/(privateProfile)/calculator/page.tsx

```diff
@@ -1,9 +1,433 @@
-function Calculator() {
+"use client";
+
+import React, { useState, useEffect } from "react";
+
+import { AiFillInfoCircle } from "react-icons/ai";
+import { CgClose } from "react-icons/cg";
+
+import style from "./calculator.module.scss";
+
+export default function Calculator() {
+	const [value, setValue] = useState<number>(0);
+	const [isVisibleInfo, setIsVisibleInfo] = useState(false);
+	const [isMobile, setIsMobile] = useState(window.innerWidth <= 425);
+
+	const formatNumber = (num: number) => {
+		return num.toString().replace(/\B(?=(\d{3})+(?!\d))/g, " ");
+	};
```

Comment:
const formatCalculateNumber = -/-

can you move this util function to separate file in src/utils folder?